tests/bluetooth/df/connectionless_cte_chains/src/common.c

```diff
@@ -171,9 +178,20 @@ void common_create_per_adv_chain(struct ll_adv_set *adv_set, uint8_t pdu_count)
 		zassert_not_null(pdu_new, "Cannot allocate new PDU.");
 		/* Initialize new empty PDU. Last AUX_CHAIN_IND may not include AuxPtr. */
 		if (idx < pdu_count - 1) {
-			ull_adv_sync_pdu_init(pdu_new, ULL_ADV_PDU_HDR_FIELD_AUX_PTR);
+			if (IS_ENABLED(CONFIG_BT_CTLR_ADV_PERIODIC_ADI_SUPPORT) &&
+			    adi_in_sync_ind == true) {
+				ull_adv_sync_pdu_init(pdu_new, ULL_ADV_PDU_HDR_FIELD_AUX_PTR |
+								       ULL_ADV_PDU_HDR_FIELD_ADI);
+			} else {
+				ull_adv_sync_pdu_init(pdu_new, ULL_ADV_PDU_HDR_FIELD_AUX_PTR);
+			}
 		} else {
-			ull_adv_sync_pdu_init(pdu_new, 0);
+			if (IS_ENABLED(CONFIG_BT_CTLR_ADV_PERIODIC_ADI_SUPPORT) &&
+			    adi_in_sync_ind == true) {
+				ull_adv_sync_pdu_init(pdu_new, ULL_ADV_PDU_HDR_FIELD_ADI);
+			} else {
+				ull_adv_sync_pdu_init(pdu_new, 0);
```

Comment:
Perhaps outside the scope of this PR, but often (at least in the host), we have `#define`s for "NONE" values, e.g. 
```suggestion
				ull_adv_sync_pdu_init(pdu_new, ULL_ADV_PDU_HDR_FIELD_NONE);
```
To indicate what "group" of `#define`s this value represent when not using an enum.

Reply:
Added `ULL_ADV_PDU_HDR_FIELD_NONE`, but I didn't change it only in the `common.c` source file. I propose to do the change in controller code as separate enhancement PR.